File: Tutorial5-1TableRelationsOneToMany/src/main/java/com/test/tutorial5_1tablerelations/data/AppDatabase.java
package com.test.tutorial5_1tablerelations.data;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

/**
 * Entities are tables in database
 */
@Database(entities = {User.class, Pet.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
    public abstract UserDao userDao();
}

File: Tutorial5-1RelationsOneToMany/src/main/java/com/test/tutorial5_1tablerelations/data/AppDatabase.kt
package com.test.tutorial5_1tablerelations.data

import androidx.room.Database
import androidx.room.RoomDatabase

/**
 * Entities are tables in database
 */
@Database(entities = [User::class, Pet::class], version = 1)
abstract class AppDatabase : RoomDatabase() {
    abstract fun userDao(): UserDao
}

